servers/src/test/java/tachyon/master/file/FileSystemMasterPermissionCheckTest.java

```diff
@@ -0,0 +1,443 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master.file;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+
+import com.google.common.collect.Lists;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+
+import tachyon.Constants;
+import tachyon.TachyonURI;
+import tachyon.conf.TachyonConf;
+import tachyon.exception.AccessControlException;
+import tachyon.exception.InvalidPathException;
+import tachyon.master.MasterContext;
+import tachyon.master.block.BlockMaster;
+import tachyon.master.file.options.CreateOptions;
+import tachyon.master.file.options.MkdirOptions;
+import tachyon.master.journal.Journal;
+import tachyon.master.journal.ReadWriteJournal;
+import tachyon.security.authentication.PlainSaslServer;
+import tachyon.security.group.GroupMappingService;
+import tachyon.thrift.FileInfo;
+
+/**
+ * Unit test for {@link tachyon.master.file.FileSystemMaster} when permission check is enabled by
```

Comment:
import tachyon.master.file.FileSystemMaster and remove the full namespace from the link